schema/schema.sql

```diff
@@ -20,36 +20,36 @@ DROP TABLE IF EXISTS `lottopool`.`agency`;
 /* Make tables */
 
 CREATE TABLE `lottopool`.`users`(
-    `id` INT NOT NULL AUTO_INCREMENT,
+    `id` UUID NOT NULL,
     `auth0_id` VARCHAR(255) NOT NULL,
     PRIMARY KEY (`id`)
 ) ENGINE=InnoDB CHARSET=utf8 COLLATE=utf8_general_ci;
 
 CREATE TABLE `lottopool`.`agency`(
-    `id` INT NOT NULL AUTO_INCREMENT,
+    `id` UUID NOT NULL,
     `name` BIGINT NOT NULL,
```

Comment:
Don't know how this one passed by me. Should be VARCHAR.

Reply:
the agency id? wouldn't this be an internal id set by us?

Reply:
The ID is fine, it's `name` that I'm concerned about. Preview window isn't the most specific, lol.

Reply:
ohhhh of course 😂